notebooks/276-stable-diffusion-torchdynamo-backend/276-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -145,14 +145,14 @@
     "## OpenVINO TorchDynamo backend\n",
     "[back to top ⬆️](#Table-of-contents:)\n",
     "\n",
-    "The [OpenVINO TorchDynamo backend](https://docs.openvino.ai/2023.3/pytorch_2_0_torch_compile.html) lets you enable [OpenVINO](https://docs.openvino.ai/2023.3/home.html) support for PyTorch models with minimal changes to the original PyTorch script. It speeds up PyTorch code by JIT-compiling it into optimized kernels. By default, Torch code runs in eager-mode, but with the use of torch.compile it goes through the following steps:\n",
+    "The [OpenVINO TorchDynamo backend](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html) lets you enable [OpenVINO](https://docs.openvino.ai/2024/home.html) support for PyTorch models with minimal changes to the original PyTorch script. It speeds up PyTorch code by JIT-compiling it into optimized kernels. By default, Torch code runs in eager-mode, but with the use of torch.compile it goes through the following steps:\n",
     "1. Graph acquisition - the model is rewritten as blocks of subgraphs that are either:\n",
     "   - compiled by TorchDynamo and “flattened”,\n",
     "   - falling back to the eager-mode, due to unsupported Python constructs (like control-flow code).\n",
     "2. Graph lowering - all PyTorch operations are decomposed into their constituent kernels specific to the chosen backend.\n",
     "3. Graph compilation - the kernels call their corresponding low-level device-specific operations.\n",
     "\n",
-    "Select device for inference and enable or disable saving the optimized model files to a hard drive, after the first application run. This makes them available for the following application executions, reducing the first-inference latency. Read more about available [Environment Variables options](https://docs.openvino.ai/2023.3/pytorch_2_0_torch_compile.html#environment-variables)"
+    "Select device for inference and enable or disable saving the optimized model files to a hard drive, after the first application run. This makes them available for the following application executions, reducing the first-inference latency. Read more about available [Environment Variables options](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html#environment-variables)"
```

Comment:
```suggestion
    "Select device for inference and enable or disable saving the optimized model files to a hard drive, after the first application run. This makes them available for the following application executions, reducing the first-inference latency. Read more about available [Environment Variables options](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html#options)"
```

Environment Variables were changed from it's own part of an article into a note, linking now "Options" which describe them in current release